### DISCUSSION TITLE
FAQ - Gold Bar gives only 100 gold coins back when smelting back at the forge

### DISCUSSION THREAD

**Discussion by dtimin** [Digitalroot-Valheim/Digitalroot.Valheim.GoldBars#7] [Q&A] [answered]
### What happened?

1. Smelt a gold bar
2. Try to smelt it back at the forge
3. It costs 1 gold, 1 copper and gives only 100 coins back, instead of 200 spent initially.
4. Should give 200 coins

### To Reproduce

1. Smelt a gold bar
2. Try to smelt it back at the forge
3. It costs 1 gold, 1 copper and gives only 100 coins back, instead of 200 spent initially.
4. Should give 200 coins

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Digitalroot** [comment, marked as answer]
This is by design. I could make the return value configurable.
